interface/main/finder/dynamic_finder_ajax.php

```diff
@@ -16,6 +16,67 @@
 require_once("../../globals.php");
 require_once($GLOBALS['srcdir']."/options.inc.php");
 
+
+//Function do translate from another language to English
+if (!(function_exists('ltox'))) {
+
+    /**
+     * @param $wordInLang
+     * @param bool $oneWord
+     * @return bool or English text of var
+     */
+    function ltox( $wordInLang, $oneWord = true )
+    {
```

Comment:
It looks like this function is being used to map gender. Wouldn't it be better to make this a selector on the search (ie. none, male, female); then no need to try to map a translation backwards and lowercase things?

Reply:
Hi. @bradymiller your are right. But i think that if somebody use a he's native language: Hebrew or Spanish languages for search by this language, in this case 
we will write "masculino" (male in Spain) in "Search all columns" input, after file will make a query with this searched word, in query we get "sex like 'masculino%'", but in table we are have values: "male" or "female".

Reply:
But the selector would have translated labels while the values would not be translated and allow mapping to the item in the database (this is one of the big advantages to having the translated label and the database "token" being separate in the layout/list/LBF stuff).

Reply:
hi @yurige-matrix ,
Why are we not able to use a selector for gender as discussed in above post?

Reply:
hi @sharonco-matrix ,
Still awaiting this comment to be addressed. Thanks, -brady

Reply:
Hi @bradymiller .Not quite clear.

Reply:
Hi @yurige-matrix ,
For the gender example, if the search were a selector (with gender options) rather than free text, then a user could just simply select the gender from the selector (which would be translated to the users language). All the information (the list) to do this is stored in the layout_options entry for gender.

Reply:
Now it is clear. In this case, you need to change the interface and search code again. I like in this case to make a query that does the search according to what is displayed on the screen.
What do you think?

Reply:
Hi @bradymiller. Changes are made to the ltox function, now the data is taken from the list_options table.